ksqldb-streams/src/test/java/io/confluent/ksql/execution/streams/SourceBuilderV1Test.java

```diff
@@ -1108,7 +1105,12 @@ private void givenWindowedSourceTableV1(final int pseudoColumnVersion) {
     );
   }
 
-  private void givenUnwindowedSourceTableV1(
+  private void givenWindowedSourceTable() {
+    givenWindowedSourceTable(ROWPARTITION_ROWOFFSET_PSEUDOCOLUMN_VERSION);
+  }
+
+
+    private void givenUnwindowedSourceTableV1(
```

Comment:
nit: whitespace change.

Reply:
I was actually referring to the fact that the indentation of this line is off but that works too 😆